src/effects/backends/builtin/glitcheffect.cpp

```diff
@@ -0,0 +1,121 @@
+#include "effects/backends/builtin/glitcheffect.h"
+
+#include <QtDebug>
+
+#include "util/math.h"
+#include "util/sample.h"
+
+QString GlitchEffect::getId() {
+    return "org.mixxx.effects.glitch";
+}
+
+EffectManifestPointer GlitchEffect::getManifest() {
+    EffectManifestPointer pManifest(new EffectManifest());
+
+    pManifest->setAddDryToWet(false);
+    pManifest->setEffectRampsFromDry(false);
+
+    pManifest->setId(getId());
+    pManifest->setName(QObject::tr("Glitch"));
+    pManifest->setShortName(QObject::tr("Glitch"));
+    pManifest->setAuthor("The Mixxx Team");
+    pManifest->setVersion("1.0");
+    pManifest->setDescription(
+            QObject::tr("Periodically samples and repeats a small portion of "
+                        "audio to create a glitchy metallic sound."));
+
+    EffectManifestParameterPointer delay = pManifest->addParameter();
+    delay->setId("delay_time");
+    delay->setName(QObject::tr("Time"));
+    delay->setShortName(QObject::tr("Time"));
+    delay->setDescription(QObject::tr(
+            "Delay time\n"
+            "1/8 - 2 beats if tempo is detected\n"
+            "1/8 - 2 seconds if no tempo is detected"));
+    delay->setValueScaler(EffectManifestParameter::ValueScaler::Linear);
+    delay->setUnitsHint(EffectManifestParameter::UnitsHint::Beats);
+    delay->setRange(0.0, 0.5, 2.0);
```

Comment:
Please define a `constexpr double maxDelay = 2.0` in the class and refer to it here instead of the literal `2.0`.